rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/db/sql/DBSQLFile.kt

```diff
@@ -4,12 +4,13 @@ import com.smeup.rpgparser.interpreter.DBFile
 import com.smeup.rpgparser.interpreter.Field
 import com.smeup.rpgparser.interpreter.Record
 import com.smeup.rpgparser.interpreter.Value
-import java.lang.RuntimeException
 import java.sql.Connection
 import java.sql.ResultSet
 
 class DBSQLFile(private val name: String, private val connection: Connection) : DBFile {
     private var resultSet: ResultSet? = null
+    private var lastKey: List<Field> = emptyList()
```

Comment:
is this specific for the readEqual? Or will this be updated by multiple operations?

Reply:
Good question. The answer is "Since I'm learning RPG while creating the interpreter, I don't really know" :-) 
Should we write the code differently depending on this?

Reply:
no, just maybe a more specific name like `readEqualLastKey`